src/psyclone/dynamo0p3.py

```diff
@@ -2866,7 +2866,8 @@ def initialise(self, parent):
                                          rhs=name + "%" +
                                          arg.ref_name(function_space) +
                                          "%get_undf()"))
-
+            # If self._dofs_only is True and dist_mem is not enabled then we
```

Comment:
Ah! That was a note-to-self about a bug I discovered. I'll take a look.

Reply:
It must have been a bug I introduced when I tried altering Builtins so that they accepted operator arguments. I gave up on that approach and have checked and this isn't an issue anymore so I've removed the comment.

Reply:
Great!